codebuild_ECR_push/code_build_ecr_container.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in to Amazon ECR...
+      - aws ecr get-login-password --region ${AWS_DEFAULT_REGION} | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
+      - echo Logging into Dockerhub...
+      - docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
```

Comment:
I appreciate that these lines were also in `cyber-security-authenticate`, but I'm not sure why we need to log in to DockerHub as well as ECR

Reply:
... it's because we need to pull from Docker Hub as part of building the Concourse Base Image, duh